content/smart.md

```diff
@@ -154,7 +156,17 @@ SMART apps can be embedded and launched from the *MPages* Workflow or Summary. I
 
 #### iframe Cookies ####
 
-If your SMART app uses cookies, the [P3P](https://www.w3.org/P3P/) header or meta tag must be set in order for cookies to be saved correctly while running in an *MPages* iframe.  Please see [this page](https://blogs.msdn.microsoft.com/ieinternals/2013/09/17/a-quick-look-at-p3p/) for more information on P3P.
+For IE browser if your SMART app uses cookies, the [P3P](https://www.w3.org/P3P/) header or meta tag must be set in order for cookies to be saved correctly while running in an *MPages* iframe.  Please see [this page](https://blogs.msdn.microsoft.com/ieinternals/2013/09/17/a-quick-look-at-p3p/) for more information on P3P.
+
+For Edge browser if your SMART app uses cookies, the `Set-Cookie` response header has to contain the attributes: `SameSite=None; Secure` for your cookies to be saved correctly while running in an iframe.
+
+### Microsoft Edge Browser (Chromium based) Support ###
```

Comment:
the Microsoft branding guidelines word it as "Microsoft Edge (Chromium based) browser" for the full reference (and "Microsoft Edge" or just "Edge" in other places)

Reply:
I removed the wording 'Edge browser' and just kept 'Edge' or 'Microsoft Edge'.